src/EventCell.js

```diff
@@ -67,7 +69,9 @@ class EventCell extends React.Component {
         selected
       )
 
+    let _draggable = get(event, draggableAccessor)
     let wrapperProps = {
+      draggable: _draggable === null ? true : _draggable,
```

Comment:
`null` is a bit wonky here. I'd say go with `draggableAccessor ? !!get(event, draggableAccessor) : true`

Reply:
I've used it because your `get` (from accessors) returns `null` in this case.

Speaking for the logic, if `get(event, draggableAccessor)` is `null`, draggable should be `true` (to enable dragging by default). So if I do `!!` , it will result as `false`, which is not what we want here :)

If you want to keep your syntax, I can define default `draggableAccessor: null` in Calendar `defaultProps`, 
or I can set it to `draggableAccessor: () => true`,  then we'll just need 
```
let wrapperProps = {
  draggable: get(event, draggableAccessor),
```
But everyone will have to set the props `draggableAccessor`  as a function  to use it

Let me know what you prefer :)

Reply:
> So if I do !! , it will result as false

Yeah i think that makes sense if you provide a draggableAccessor the intent is you want to determine if it's draggable, i'd prefer to split out the distinction between "I returned a falsy value from draggableAccessor" and "I did not provide a draggableAccessor", I think it makes sense to default true if they didn't provide it, and if they did assume the return value indicates  something.

Reply:
sounds good!